### DISCUSSION TITLE
英雄狗熊只在一線之隔

### DISCUSSION THREAD

**Discussion by 1104030360** [1101-GS4719-C/Python-Community-GS4719-C#1372] [General]
前天有在關注NBA的人應該知道在湖人跟勇士季後賽第一戰最後讀秒階段，勇士隊的新星Jordan poole投出了一記大號三分球，遺憾的是球並未投進，隨之而來的是球迷的龐大質疑，為什麼不把球在分出去重新組織，為什麼不選擇切入更保險，原本的我也如同那些球迷依樣報有同樣的疑問，但是靜下心來仔細回想才知道Jordan poole在前面幾節其實正是勇士隊能把比分緊咬的重要關鍵，他的外線手感非常火熱，然而在最後一刻失手了卻變成了許多球迷所認為的戰敗首因，我認為這是不公平的，或許他的出手不是最佳選擇，但是依照他那天火燙的手感，他來終結比賽也是合理的，我認為這就是生活的現實面，常常在重要關鍵必須要有人跳出來做出關鍵決定，如果成功了他會被大家捧上天，相反的，失敗了會被罵到臭頭，以前的我會在關鍵時刻猶豫不決因為怕被大家責罵，但是現在的我不會了，因為我覺得在關鍵時刻做出合理的判斷卻失敗成了狗熊並不可恥，甚至我認為他很棒！

**Comment by tower20021202** [comment]
我認為那場全隊除了咖哩被守死，當天最準的就是poole了，沒有他勇士比分早就被拉開了，況且最後時間也快不夠了，當天手感火燙的他，在最後空擋大號三分出手並非一個差的選擇，就只是沒進而以，但在關鍵時刻能不能帶領球隊走向勝利，應該就是poole目前跟巨星的差別吧
